contracts/implementations/RMRKEquippableImpl.sol

```diff
@@ -76,7 +76,7 @@ contract RMRKEquippableImpl is
         uint256 mintPriceRequired = numToMint * _pricePerMint;
         if (mintPriceRequired != msg.value) revert RMRKMintUnderpriced();
 
-        uint256 nextToken = _totalSupply + 1;
+        uint256 nextToken = _totalSupply + 1; // Doesn't this cause gaps in the IDs? So for example, if this would be the first mint ever on a collection, it would set the initial ID of the token to 1 instead of 0.
```

Comment:
We don't want people to use tokenId 0, it will mess up in some validations. Maybe we should be more explicit about this or even enforce it somewhere (maybe inside `_mint`?)

Reply:
We should definitely emphasise this somewhere. Maybe examples first and then in the EVM docs as well (once we have them)? 🤔

Reply:
Actually we need discourage 0 in general for ids. On resources, base parts, equippable groups. Will take note.

Reply:
Yeah, I'm pretty sure the majority of engineers will try to start with ID 0. We need to enforce this in some manner.

Reply:
I agree with this point, starting from id = 1 should be pointed out both in the docs and the examples, especially because the rest of the  NFT industry uses token ids beginning from 0.

Reply:
Could we give a special status to ID 0? Like a representation of the collection? So NFT with ID 0 wouldn't be a regular NFT, but rather a 'cover' NFT of the collection? 🤔

Reply:
The thing with Id 0 is that we use a lot of mappings where Ids are either keys or in values. When querying for something if the value is 0 we assume there's no entry. So having a custom logic for 0 is probably too complex.
For 'cover' or collection data we already have collection meta data (extension).